core/src/main/java/org/apache/struts2/StrutsConstants.java

```diff
@@ -230,6 +230,8 @@ public final class StrutsConstants {
     public static final String STRUTS_XWORKCONVERTER = "struts.xworkConverter";
 
     public static final String STRUTS_ALWAYS_SELECT_FULL_NAMESPACE = "struts.mapper.alwaysSelectFullNamespace";
+    /** Disable fallback to empty namespace when request namespace didn't match any in action configuration */
+    public static final String STRUTS_DISABLE_EMPTY_NAMESPACE_FALLBACK = "struts.disableActionConfigFallbackToEmptyNamespace";
```

Comment:
@lukaszlenart Any thoughts on what the best name for this constant might be?

Reply:
Wouldn't be better to have `struts.actionConfig.fallbackToEmptyNamespace` with default value set to `true` instead?

Reply:
Sounds reasonable to me - @jefferyxhy could you please update?

Reply:
@kusalk @lukaszlenart updated. Thanks